Anlab.Mvc/Client/components/ClientId.tsx

```diff
@@ -8,12 +8,14 @@ export interface IClientInfo {
     employer: string;
     name: string;
     email: string;
+    copyEmail?: string;
+    subEmail?: string;
     phoneNumber: string;
 }
 
 interface IClientIdProps {
     //change one property of parent ClientInfo state
-    handleClientInfoChange: (key: string, value: string) => void;
+    handleClientInfoChange: (key: string[], value: string[]) => void;
```

Comment:
@laholstege make a quick change to the comment since we are updating >=1 prop, and pluralize the prop names